src/core/style/expressions/binary.js

```diff
@@ -2,24 +2,328 @@ import { float } from '../functions';
+ * const style = new carto.Style({
+ *  width: s.floatSub(10, 2);  // Upon rendering, width will be evaluated internally to 8
+ * });
+ *
+ * @memberof carto.style.expressions
+ * @name floatSub
+ * @function
+ * @api
+ */
 export const FloatSub = genBinaryOp((x, y) => x - y, (x, y) => `(${x} - ${y})`);
+
+/**
+ *
+ * Modulus of two numeric expressions, mod returns a numeric expression with the value of x modulo y. This is computed as x - y * floor(x/y).
+ *
+ * @param {carto.style.expressions.Expression | number} x numeric expression
+ * @param {carto.style.expressions.Expression | number} y numeric expression
+ * @return {carto.style.expressions.Expression} numeric expression
+ *
+ * @example <caption>Number modulus.</caption>
+ * const s = carto.style.expressions;
+ * const style = new carto.Style({
+ *  width: s.mod(10, 6);  // 4
+ * });
+ *
+ * @memberof carto.style.expressions
+ * @name floatMod
+ * @function
+ * @api
+ */
 export const FloatMod = genBinaryOp((x, y) => x % y, (x, y) => `mod(${x}, ${y})`);
+
+/**
+ *
+ * Compute the base to the exponent power, return a numeric expression with the value of the first parameter raised to the power of the second.
+ * The result is undefined if x<0 or if x=0 and y≤0.
+ *
+ * @param {carto.style.expressions.Expression | number} base numeric expression
+ * @param {carto.style.expressions.Expression | number} exponent numeric expression
+ * @return {carto.style.expressions.Expression} numeric expression with base ^ exponent
+ *
+ * @example <caption>Power of two numbers.</caption>
+ * const s = carto.style.expressions;
+ * const style = new carto.Style({
+ *  width: s.floatPow(2, 3);  // 8
+ * });
+ *
+ * @memberof carto.style.expressions
+ * @name floatPow
+ * @function
+ * @api
+ */
 export const FloatPow = genBinaryOp((x, y) => Math.pow(x, y), (x, y) => `pow(${x}, ${y})`);
 
+/**
+ *
+ * Compare if x is greater than y.
+ *
+ * This returns a numeric expression where 0 means `false` and 1 means `true`.
+ *
+ * @param {carto.style.expressions.Expression | number} x numeric expression
+ * @param {carto.style.expressions.Expression | number} y numeric expression
+ * @return {carto.style.expressions.Expression} numeric expression
+ *
+ * @example <caption>Compare two numbers to show only elements with price > 30</caption>
+ * const s = carto.style.expressions;
+ * const style = new carto.Style({
+ *  filter: s.gt(s.property('price'), 30);
+ * });
+ *
+ * @memberof carto.style.expressions
+ * @name gt
+ * @function
+ * @api
+ */
 export const GreaterThan = genBinaryOp((x, y) => x > y ? 1 : 0, (x, y) => `(${x}>${y}? 1.:0.)`);
+
+/**
+ *
+ * Compare if x is greater than or equal to y.
+ *
+ * This returns a numeric expression where 0 means `false` and 1 means `true`.
+ *
+ * @param {carto.style.expressions.Expression | number} x numeric expression
+ * @param {carto.style.expressions.Expression | number} y numeric expression
+ * @return {carto.style.expressions.Expression}
```

Comment:
I think we can assume that comparison operations will return a `float` expression